features/FEATURE_BLE/ble/pal/SimpleEventQueue.h

```diff
@@ -73,8 +73,8 @@ struct SimpleEventQueue : EventQueue {
         if (_ble_base == NULL) {
             return false;
         }
-
-        EventNode* next = new (std::nothrow) EventNode(event);
+        uint8_t* event_buf = (uint8_t*)WsfBufAlloc(sizeof(EventNode));
```

Comment:
- The `NULL` check should be made after `WsfBufAlloc`.
- placement new accepts a `void*`. The cast to `uint8_t` is not required.